h/buildext.py

```diff
@@ -218,45 +187,84 @@ def build_chrome(args):
     copytree('h/browser/chrome/help', 'build/chrome/help')
     copytree('h/browser/chrome/images', 'build/chrome/images')
     copytree('h/static/images', 'build/chrome/public/images')
+    copytree('h/static/styles/vendor/fonts', 'build/chrome/public/fonts')
 
-    os.makedirs('build/chrome/lib')
+    subprocess_args = ['node_modules/.bin/gulp', 'build']
+    subprocess.call(subprocess_args)
 
-    subprocess_args = ['node_modules/.bin/browserify',
-                       'h/browser/chrome/lib/extension.js', '--outfile',
-                       'build/chrome/lib/extension-bundle.js']
+    os.makedirs('build/chrome/lib')
+    shutil.copyfile('build/scripts/extension.bundle.js',
+                    'build/chrome/lib/extension.bundle.js')
     if args.debug:
-        subprocess_args.append('--debug')
-    subprocess.call(subprocess_args)
+        shutil.copyfile('build/scripts/extension.bundle.js.map',
+                        'build/chrome/lib/extension.bundle.js.map')
+
+    webassets_env = AssetEnvironment('/public')
+    webassets_env.register('app_js', [
```

Comment:
I'm pretty sure you could just use `webassets` itself without too much trouble. Just use `webassets` directly without using the `pyramid_webassets` indirection layer that the application uses.

And yes, if possible let's stick with reading `h/assets.yaml`. I'm probably not in favour of switching it to JSON -- configuration files need comments and JSON doesn't have comments.